ansible/README.md

```diff
@@ -23,12 +23,19 @@ EOF
 
 ### dependencies
 
+#### Ubuntu
 apt-get should now work to install the packages needed to use ansible.
 
-```
+```bash
```

Comment:
Looks like a number of the image format sections ``` bash not correct. Some of them were in the file before being updated. Can you please fix these too.

Each ``` section needs to have a language type and I think it must have a space before the language type.

 Each code section needs to be something like this ``` bash with a space.